sklearn/metrics/classification.py

```diff
@@ -1530,6 +1530,12 @@ class 2       1.00      0.67      0.80         3
     else:
         labels = np.asarray(labels)
 
+    # labelled micro average
+    label_micro_average = False
```

Comment:
It might be simpler to do this without `if`:

```
label_micro_average = (y_type == 'multiclass' and
                       (not labels_given or ...))
```

Reply:
"label_micro_average" is a confusing name. Perhaps `micro_is_accuracy`?

Reply:
done